dotcom-rendering/src/palette.ts

```diff
@@ -1615,6 +1615,14 @@ const subNavBorder = ({ design, theme }: ArticleFormat): string => {
 			}
 	}
 };
+const subNavLink = (format: ArticleFormat) => {
+	switch (format.design) {
```

Comment:
FYI, you can also type the `subNavLink` variable direcly:

```suggestion
const subNavLink: PaletteFunction = ({ design }) => {
	switch (design) {
```

Reply:
ooo thanks for this, I didn't know that